mylar/PostProcessor.py

```diff
@@ -2844,6 +2844,10 @@ def sendnotify(self, series, issueyear, issuenumOG, annchk, module, imageFile):
             slack = notifiers.SLACK()
             slack.notify("Download and Postprocessing completed", prline2, module=module)
 
+        if mylar.CONFIG.DISCORD_ENABLED:
+            discord = notifiers.SLACK()
```

Comment:
Think this is supposed to be ```notifiers.DISCORD()``` and not ```notifies.SLACK()```

Reply:
Whoops, missed that one. Should be good now.